docs/reference/9-analytics-middle-tier/amt-overview.md

```diff
@@ -0,0 +1,229 @@
+---
+sidebar_position: 1
+---
+
+# AMT Overview
+
+## Introduction
+
+The Analytics Middle Tier (AMT) is a collection of SQL views that enable data
+analysts to better understand and access data in an Ed-Fi ODS / API while
+building reports and analytics.
+
+:::warning
+
+The sections below provide the historical context and functional analysis that
+led to the initial development of the AMT in 2018.
+
+:::
+
+## Background
+
+### Problem Definition
+
```

Comment:
The Background heading here is empty and the same level as the Problem Definition tag below. I'm think making Background level 1 is the easiest adjustment to make it consistent.

Reply:
Problem definition is now at third level. Can only have one first level per document. This now has the expected table of contents.

![image](https://github.com/user-attachments/assets/1739ebfd-7460-4445-88eb-02ff7023fe9d)

Reply:
I see this was from the old version I had checked out